content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: One Year of Excalidraw
+date: 2020-12-25
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Excalidraw started as a way to procrastinate on January 1st of 2020 and ended up being a fully fledged product only one year later! It unfortunately became a core part of many people's workflows due to the pandemic. In this post, we'll go over the most important features that made Excalidraw what it is.
```

Comment:
> It unfortunately became a core part of many people's workflows due to the pandemic.

I understand the sentiment, but when you say "something unfortunately became your workflow", it generally means the workflow is unfortunate, not the cause.